sample_rock/rockcraft.yaml

```diff
@@ -11,37 +11,21 @@ platforms:
   amd64:
 
 parts:
-  flask-environment:
-    plugin: nil
-    overlay-script: |
-      mkdir $CRAFT_OVERLAY/etc
-      mkdir $CRAFT_OVERLAY/tmp
-      chmod 755 $CRAFT_OVERLAY/etc
-      chmod 777 $CRAFT_OVERLAY/tmp
-      groupadd -R $CRAFT_OVERLAY --gid 2000 flask
-      useradd -R $CRAFT_OVERLAY --system --gid 2000 --uid 2000 --home /srv/flask flask
   flask-dependencies:
+    plugin: python
+    source: .
     stage-packages:
-      - python3-dev
+      - python3-venv
       - python3-gunicorn
```

Comment:
I think that in the latest discussion of 12-factor we decided that we will use pip for gunicorn and flask because the deb packages are lagging behind and that could cause pb
So I think we can remove this and install flask and gunicorn through pip install

Reply:
That will surely simplify things

Reply:
Fixed this